curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f039ff313dbb696b007ca.md

```diff
@@ -13,7 +13,7 @@ When an array holds values, or <dfn>elements</dfn>, those values are separated b
 let array = ["first", "second"];
 ```
 
-Change your `rows` declaration to be an array with the strings `Naomi`, `Quincy,` and `CamperChan`. The order of values in an array is important, so follow that order. Remember that strings are case-sensitive.
+Change your `rows` declaration to be an array with the strings `Naomi`, `Quincy` and `CamperChan`. The order of values in an array is important, so follow that order. Remember that strings are case-sensitive.
```

Comment:
there can still be a comma here.
It just needs to be outside the backticks.

```suggestion
Change your `rows` declaration to be an array with the strings `Naomi`, `Quincy`, and `CamperChan`. The order of values in an array is important, so follow that order. Remember that strings are case-sensitive.
```

Reply:
Made the changes.
Please review.